portal/templates/initial_queries.html

```diff
@@ -1,7 +1,7 @@
 {% extends "layout.html" %}
+{%- block mainclass -%}class="vis-on-callback"{%- endblock -%}
```

Comment:
Would it make sense to use the [VueJS `data` object](https://vuejs.org/v2/guide/state-management.html#Simple-State-Management-from-Scratch) to store this state instead of the DOM, or would that be overkill (eg VueJS not used in this page)?

Reply:
yeah, that is a good idea.  but you are right VueJS is not used on this page yet but I plan on converting it to using it if I have downtime.